### ISSUE TITLE
Keras ResNet50 model loaded with load_model( ) with no softmax layer

### ISSUE THREAD

**Issue by CarlosE97** [albermax/innvestigate#148]
Hello @albermax, I've got a ResNet50 model's .h5 which I'm loading using the keras' load_model("path"). But I'm encountering an Exception:

> File "imagenet_compare_networks.py", line 128, in <module>
    tmp = prepare_model_data_and_analyzers(netname)
  File "imagenet_compare_networks.py", line 39, in prepare_model_data_and_analyzers
    net = tmp(load_weights=True, load_patterns="relu")
  File "/usr/local/lib/python3.6/site-packages/innvestigate/applications/imagenet.py", line 343, in resnet50customv2
    is_loaded_model=True)
  File "/usr/local/lib/python3.6/site-packages/innvestigate/applications/imagenet.py", line 155, in _prepare_keras_net
    net["out"] = kgraph.pre_softmax_tensors(model.outputs)
  File "/usr/local/lib/python3.6/site-packages/innvestigate/utils/keras/graph.py", line 375, in pre_softmax_tensors
    raise Exception("No softmax found.")
Exception: No softmax found.

Obviously, my model has no softmax layer. What can be done in this case? 

Thanks

**Comment by albermax** [comment]
Hi Carlos,

strange. Does your model actually have no softmax?
If so why do you use the function pre_softmax_tensors?

Cheers,
Max

**Comment by CarlosE97** [comment]
Hello @albermax, I followed your instructions and I didn't use the pre_softmax_tensors and it worked out although I never got a reasonable "probability" which is pretty much logical since the model lacks the softmax layer.
Thanks for the help

Best,
Carlos
